src/@eggheadio/gatsby-theme-egghead-blog/templates/blog.js

```diff
@@ -0,0 +1,158 @@
+
+  return (
+    <Layout site={site}>
+      <SEO />
+      <Container noVerticalPadding>
+        {posts.map(({ node: post }) => (
+          <div
+            key={post.id}
+            css={css`
+              :not(:first-of-type) {
+                margin-top: 60px;
+                ${bpMaxMD} {
+                  margin-top: 40px;
+                }
+                ${bpMaxSM} {
+                  margin-top: 20px;
+                }
+              }
+              :first-of-type {
+                margin-top: 20px;
+                ${bpMaxSM} {
+                  margin-top: 20px;
+                }
+              }
+              .gatsby-image-wrapper {
+              }
+              ${bpMaxSM} {
+                padding: 20px;
+              }
+              display: flex;
+              flex-direction: column;
+            `}
+          >
+            {post.frontmatter.banner && (
+              <div
+                css={css`
+                  ${bpMaxSM} {
+                    padding: 20px;
+                  }
+                `}
+              >
+                <Link
+                  aria-label={`View ${post.frontmatter.title} article`}
+                  to={`/${post.fields.slug}`}
+                >
+                  <Img sizes={post.frontmatter.banner.childImageSharp.fluid} />
+                </Link>
+              </div>
+            )}
+            <h2
+              css={css`
+                margin-top: 30px;
+                margin-bottom: 10px;
+              `}
+            >
+              <Link
+                aria-label={`View ${post.frontmatter.title} article`}
+                to={`/${post.fields.slug}`}
+              >
+                {post.frontmatter.title}
+              </Link>
+            </h2>
+            {/* <small>{post.frontmatter.date}</small> */}
+            <p
+              css={css`
+                margin-top: 10px;
+              `}
+            >
+              {post.excerpt}
+            </p>{' '}
+            <Link
+              to={`/${post.fields.slug}`}
+              aria-label={`view "${post.frontmatter.title}" article`}
+            >
+              Read Article →
+            </Link>
+          </div>
+        ))}
+        <div css={css({ marginTop: '30px' })}>
+          {nextPagePath && (
+            <Link to={nextPagePath} aria-label="View next page">
+              Next Page →
+            </Link>
+          )}
+          {previousPagePath && (
+            <Link to={previousPagePath} aria-label="View previous page">
+              ← Previous Page
+            </Link>
+          )}
+        </div>
+        <hr
+          css={css`
+            margin: 50px 0;
+          `}
+        />
+      </Container>
+    </Layout>
+  )
+}
+
+export default Blog
+
+export const pageQuery = graphql`
+  query {
+    site {
+      ...site
+    }
+    allMdx(
+      sort: { fields: [frontmatter___date], order: DESC }
+      filter: {fileAbsolutePath: {regex: "//content/blog//"}}
```

Comment:
This was neat.

Current implementation from the theme was breaking for me and I saw this is how kent does it on his blog.

`filter: { fields: { isPost: { eq: true } } }`

This filter in the page query will break because of shadowing for some reason. I see in the theme that  `isPost` is a field that gets created on the node but for whatever reason when you shadow and do a page query, that created field isn't available to you. 

[Link to PR that does this](https://github.com/eggheadio/gatsby-theme-egghead-blog/commit/9e0fba5f0925c67a1e83c4cc94704a69a16115d1)